cmd/opentelemetry-collector/app/defaults/defaults.go

```diff
@@ -32,12 +29,6 @@ import (
 
 // Components creates default and Jaeger factories
 func Components(v *viper.Viper) config.Factories {
-	// Add flags to viper to make the default values available.
-	// OTEL collector creates the default configuration in service.New() to validate that
-	// factories can create config.
-	// However, at this point the Jaeger storage flags are not added to viper.
-	// The Jaeger storage flags are added to cobra and then to viper in main after service.New().
-	initViper(v)
```

Comment:
Could you just confirm what has changed so this is no longer necessary?

Reply:
See https://github.com/jaegertracing/jaeger/pull/2175#discussion_r406147014 done in https://github.com/open-telemetry/opentelemetry-collector/pull/769

Reply:
It is indeed needed. All flags have to be added to viper bc exporter can be specified in OTEL conf file and we cannot know it before the service start. 

It will be fixed in https://github.com/jaegertracing/jaeger/pull/2181/files#diff-bb35f0b5f0f17069f6a3a53b4013191cR38